### PR TITLE
Fix some tests : make coverage -> test

### PR DESCRIPTION
**Pull request by sapk**
make coverage use mysql database in paralell than test-mysql. That introduce test fails for test like GPG that insert key at runtime.  

 And some quick fix of missed error during PRs.

### CODE REVIEW

**Review by jonasfranz — CHANGES_REQUESTED** [review]
The test still fails: https://drone.gitea.io/go-gitea/gitea/499/1/test-mysql

File: modules/mailer/mailer.go [hunk 1]
```diff
@@ -42,7 +42,7 @@ func NewMessageFrom(to []string, from, subject, body string) *Message {
 
 	plainBody, err := html2text.FromString(body)
 	if err != nil || setting.MailService.SendAsPlainText {
-		if strings.Contains(base.TruncateString(body, 100), "<html>"){
+		if strings.Contains(base.TruncateString(body, 100), "<html>") {
```
Comment (jonasfranz) [inline modules/mailer/mailer.go]: Please revert a885c7f since theire is already a PR for this: #2490

**Review by jonasfranz — CHANGES_REQUESTED** [review]

File: modules/mailer/mailer.go (hunk 1 above)
Comment (sapk) [reply modules/mailer/mailer.go]: Good catch

**Comment by sapk** [comment]
@JonasFranzDEV Yes, I haven't look in detail for the part still failling. I have mostly look for the what happen aroung the GPG part since I have worked on it. ^^

**Comment by lunny** [comment]
But why so many files changed?

**Comment by lafriks** [comment]
@lunny I just moved `integrations/gitea-repositories-meta/gitea-repositories` to `integrations/gitea-repositories-meta` so that for each (mysql/pgsql/sqlite) sepearate `gitea-repositories-*` directory could be used and there is no hardcoded value but app.ini configuration value is reused

**Comment by lunny** [comment]
LGTM

**Review by appleboy — APPROVED** [review]

**Comment by appleboy** [comment]
LGTM

**Comment by sapk** [comment]
thx @lafriks

**Pull request merged**